contracts/contracts/recipientRegistry/PessimisticRecipientRegistry.sol

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.6.12;
+
+import '@openzeppelin/contracts/access/Ownable.sol';
+
+import './BaseRecipientRegistry.sol';
+
+/**
+ * @dev Recipient registry with pessimistic execution of registrations and removals.
```

Comment:
I think we should add a bit of clarity here - basically that this contract is exactly the same as `OptimisticRecipientRegistry` except that only the owner can call `executeRequest`, so only the contract owner has the ability to actually add recipients or remove recipients from the registry.

Reply:
Looks like you also removed `challengePeriodDuration`, since that's no longer relevant. I'd mention that as well.

Reply:
This is an interesting point do both contracts have the same ABI? @pettinarip

Reply:
@daodesigner I can't imagine they do given they have different functions (e.g. this contract doesn't have `setChallengePeriodDuration`).